### PR TITLE
[MODFISTO-512] Add test coverage for organization, invoice, invoice line audit evets and encumbrance amount after issuing credit

### PR DESCRIPTION
**Pull request by Saba-Zedginidze-EPAM**
## Purpose
[[MODINVOSTO-187] Implement audit outbox pattern for sending kafka events about invoice updates](https://folio-org.atlassian.net/browse/MODINVOSTO-187)
[[MODINVOSTO-188] Implement audit outbox pattern for sending kafka events about invoice line updates](https://folio-org.atlassian.net/browse/MODINVOSTO-188)
[[MODORGSTOR-164] Implement audit outbox pattern for sending kafka events about organization updates](https://folio-org.atlassian.net/browse/MODORGSTOR-164)
[[MODFISTO-512] Encumbrance value incorrect after paying credit invoice](https://folio-org.atlassian.net/browse/MODFISTO-512)

## Approach
- Enable tests for invoice/invoice-line audit events
- Refactor organization tests and add new test for audit events
- Add test scenario for fixing encumbrance amount after issuing credit for paid order

## Screenshots
image
image
image

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/mod-organizations/features/acquisitions-api-tests.feature [hunk 1]
```diff
@@ -8,145 +8,37 @@ Feature: Organizations API tests.
...
-  # --- Create API test(s) section start ---
+    * table organizationsData
+      | id                          | code                 | status   | acqUnitIds                    |
+      | noAcqOrganizationId         | 'NO_ACQ_ORG'         | 'Active' | []                            |
+      | readOnlyOrganizationId      | 'READ_ONLY_ORG'      | 'Active' | ['#(readOnlyAcqUnitId)']      |
+      | updateOnlyOrganizationId    | 'UPDATE_ONLY_ORG'    | 'Active' | ['#(updateOnlyAcqUnitId)']    |
+      | fullProtectedOrganizationId | 'FULL_PROTECTED_ORG' | 'Active' | ['#(fullProtectedAcqUnitId)'] |
+    * def v = callonce createOrganization organizationsData
 
   Scenario: Get not protected org
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-organizations/features/acquisitions-api-tests.feature:43]: We can slap here a @Negative indicator

File: acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-issuing-credit-for-paid-order.feature [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+      * def budgetId = callonce uuid2
+      * def orderId = callonce uuid3
+      * def poLineId = callonce uuid4
+      * def invoiceId1 = callonce uuid5
+      * def invoiceId2 = callonce uuid6
+      * def invoiceLineId1 = callonce uuid7
+      * def invoiceLineId2 = callonce uuid8
+
+
+    Scenario: Check the encumbrances after issuing credit when the order is fully paid
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/cross-modules/features/check-encumbrances-after-issuing-credit-for-paid-order.feature:27]: This and other response code 200-204 scenarios can be annotated with a @Positive indicator

File: acquisitions/src/main/resources/thunderjet/mod-organizations/features/audit-event-organization.feature [hunk 3]
```diff
@@ -0,0 +1,78 @@
+Feature: Audit events for Organization
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-organizations/features/audit-event-organization.feature:1]: You can attach a ticket number here too

File: acquisitions/src/main/resources/thunderjet/mod-organizations/reusable/create-acq-unit.feature [hunk 4]
```diff
@@ -0,0 +1,25 @@
+@ignore
+Feature: Create ACQ Unit
+  # parameters: id?, name, code, status?
```
Comment (BKadirkhodjaev) [inline acquisitions/src/main/resources/thunderjet/mod-organizations/reusable/create-acq-unit.feature:3]: It seems like status is not being used here?

**Comment by SerhiiNosko** [comment]
@Saba-Zedginidze-EPAM can you check please can we adjut integration test for claiming to test this scenario
https://github.com/folio-org/mod-orders-storage/pull/461
https://github.com/folio-org/folio-integration-tests/blob/master/acquisitions/src/main/resources/thunderjet/mod-orders/features/piece-batch-job.feature

**Review by BKadirkhodjaev — APPROVED** [review]

File: acquisitions/src/main/resources/thunderjet/mod-organizations/features/audit-event-organization.feature (hunk 3 above)
Comment (Saba-Zedginidze-EPAM) [reply acquisitions/src/main/resources/thunderjet/mod-organizations/features/audit-event-organization.feature:2]: Attached, thanks

File: acquisitions/src/main/resources/thunderjet/mod-organizations/reusable/create-acq-unit.feature (hunk 4 above)
Comment (Saba-Zedginidze-EPAM) [reply acquisitions/src/main/resources/thunderjet/mod-organizations/reusable/create-acq-unit.feature]: Thanks, fixed the arguments

**Pull request merged**